elixir/apps/web/test/web/live/settings/identity_providers/google_workspace/connect_test.exs

```diff
@@ -74,7 +74,7 @@ defmodule Web.Live.Settings.IdentityProviders.GoogleWorkspace.Connect do
 
       callback_url =
         url(
-          ~p"/#{account.id}/settings/identity_providers/google_workspace/#{provider.id}/handle_callback"
```

Comment:
Here, we should still use the ID to redirect from IdP to Firezone. We don't want the sync to break when the slug is changed while those URLs must be whitelisted on the IdP end.

Reply:
This is a good point.  This means I'll need to go in and update the code that does the redirect to the IDP to only use the `account.id` rather than accepting an `account` and letting the protocol handle the interpolation.  

I'll also need to make sure that what's displayed to the user for setting up their IDP is shown as a URL with the `account.id` rather than a URL with the `account.slug`.

In keeping with my previous comment, how you would feel about leaving this as is for now and me putting up a more targeted follow on PR for this?

Reply:
I think the problem here is that this PR changes it to purely use slugs in the first place so it feels not right to introduce an issue in code not related to the PR to fix it later in another one. 

Maybe let's move it out to a separate PR which will enforce slug usage everywhere except with IdP redirects? Should be pretty easy to split the code.

Reply:
Ok, that's fair.  Yeah, I can remove the code to default the URLs to use slugs and then open a new PR for just that.

Reply:
I've removed the code that defaulted all references of account to `account.slug`.  Also updated/reverted any changed tests.